pandas/tests/frame/test_combine_concat.py

```diff
@@ -347,6 +347,47 @@ def test_concat_named_keys(self):
                                              names=[None, None]))
         assert_frame_equal(concatted_unnamed, expected_unnamed)
 
+    def test_concat_axis_parameter(self):
+        # GH 14369
+        df1 = pd.DataFrame({'A': [0.1, 0.2]}, index=range(2))
+        df2 = pd.DataFrame({'A': [0.3, 0.4]}, index=range(2))
+        expected_row = pd.DataFrame(
+            {'A': [0.1, 0.2, 0.3, 0.4]}, index=[0, 1, 0, 1])
+        concatted_row = pd.concat([df1, df2], axis='rows')
+        assert_frame_equal(concatted_row, expected_row)
+
+        expected_index = pd.DataFrame(
+            {'A': [0.1, 0.2, 0.3, 0.4]}, index=[0, 1, 0, 1])
+        concatted_index = pd.concat([df1, df2], axis='index')
+        assert_frame_equal(concatted_index, expected_index)
+
+        expected_0 = pd.DataFrame(
+            {'A': [0.1, 0.2, 0.3, 0.4]}, index=[0, 1, 0, 1])
```

Comment:
redefining those expected frames is not needed. They should be the same for all different ways to specify one of the two axis. 
This will also make the test code a bit shorter